Task 13/Program.cs

```diff
@@ -0,0 +1,45 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using System.IO;
+
+namespace Task_13
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            var names = new List<string>();
+            
+            using (var stream = new StreamReader("names.txt"))
+            {
+                while (!stream.EndOfStream)
+                {
+                    var name = stream.ReadLine();
+                    if(!names.Contains(name.Substring(0,1).ToUpper()+name.Substring(1)))
+                       names.Add(name.Substring(0, 1).ToUpper() + name.Substring(1));
```

Comment:
DRY: налицо повторяемость кода. Этого надо избегать. Например, так
```suggestion
                    var name = stream.ReadLine();
                    name = name.Substring(0,1).ToUpper()+name.Substring(1);
                    if(!names.Contains(name))
                       names.Add(name);
```